tesseract_ros/tesseract_rosutils/include/tesseract_rosutils/conversions.h

```diff
@@ -0,0 +1,203 @@
+#ifndef TESSERACT_PROCESS_PLANNING_CONVERSIONS_H
+#define TESSERACT_PROCESS_PLANNING_CONVERSIONS_H
+
+#include <Eigen/Core>
+#include <Eigen/Geometry>
+#include <tesseract_planners/core/waypoint.h>
+#include <sensor_msgs/JointState.h>
+#include <trajectory_msgs/JointTrajectory.h>
+#include <eigen_conversions/eigen_msg.h>
+#include <tesseract_process_planning/process_definition.h>
+#include <tesseract_process_planning/process_planner.h>
+#include <iostream>
+#include <fstream>
+
+namespace tesseract_process_planning
```

Comment:
Should the namespace be `tesseact_rosutils`?

Reply:
yes,